hasjob/templates/detail.html

```diff
@@ -364,10 +360,12 @@
   {%- endif %}
 {% endblock %}
 {% block postcontent %}
-  <ul id="stickie-area" class="row">
+  <ul id="related-posts" class="row">
```

Comment:
It does. The related posts aren't triggering locally with archive mode, hence didn't notice. Removing it.